File: trainee-biz/trainee-example/src/main/java/com/yuxing/trainee/example/serialization/Person.java
package com.yuxing.trainee.example.serialization;

import lombok.Data;

import java.io.Serializable;

/**
 * @author yuxing
 * @since 2022/1/24
 */
@Data
public class Person implements Serializable {

    private static final long serialVersionUID = 4645918375944963622L;

    static String country = "ITALY";
    private int age;
    private String name;
    transient int height;
}

File: trainee-kotlin/src/main/kotlin/com/yuxing/trainee/kotlin/Person.kt
package com.yuxing.trainee.kotlin

/**
 *
 * @author yuxing
 * @since 2021/12/31
 */
class Person(firstName: String) {

    private val name: String

    init {
        name = firstName
    }

    fun getName(): String {
        return this.name
    }
}